v2/brigadier/src/events.ts

```diff
@@ -1,26 +1,58 @@
 import { Worker } from "./workers"
 import { Project } from "./projects"
 
+/**
+ * An event received by Brigade that resulted in triggering an event handler.
+ */
 export interface Event {
+  /** A unique identifier for the event. */
   id: string
+  /** The project that registered the handler being called for the event. */
```

Comment:
Related to [this comment](https://github.com/brigadecore/brigade/pull/1292/files?file-filters%5B%5D=.ts#r591609582)

It might be better to say something along the lines of "relevant details of the project to which this event belongs."

But this is a non-blocking nit.

Reply:
Updated the wording on the Project interface so they'll see that if they drill in; I feel "Information about..." or "Details of..." is probably clutter here (though I do see where you are coming from).

Reply:
That sounds reasonable.